.github/workflows/tests-pro-integration.yml

```diff
@@ -336,7 +336,6 @@ jobs:
           GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           DEBUG: 1
           DISABLE_BOTO_RETRIES: 1
```

Comment:
q: This is an interesting change. You are disabling the global OpenAPI checks on every single request and instead just activate it selectively for certain tests. Could you elaborate a bit on the reasoning here? Why not just enabling it for all requests and responses for the whole execution of the test pipeline? Iirc the intention for the request validator was to even globally enable it by default for our users (in order to catch 4xx errors early without having to handle them in the actual endpoint implementations).

Reply:
Good question :) I enabled this with the PR introducing the validation and the pipeline was immediately green.
However, this was a false positive due to this typo: https://github.com/localstack/localstack/blob/e00a3623d5d7ddde2f5abbbe5db817bfe6235de8/localstack-core/localstack/aws/handlers/validation.py#L137.
Such a line was raising an exception and the handler was not stopping. As I fixed this, I saw that we have some gaps for some services, so I switched to a selective approach first. Globally enabling the verification is still the goal down the line.